action.yml

```diff
@@ -14,9 +14,9 @@ inputs:
     required: false
     default: 'jdk'
   architecture:
-    description: 'The architecture of the package'
+    description: 'The architecture of the package (defaults to the action runner's architecture)'
     required: false
-    default: 'x64'
+    default: ''
```

Comment:
Could you please remove it because by default the `gitInput` will return an empty string.